mypyc/ir/rtypes.py

```diff
@@ -145,6 +143,12 @@ def accept(self, visitor: 'RTypeVisitor[T]') -> T:
     def serialize(self) -> str:
         return 'void'
 
+    def __eq__(self, other: object) -> bool:
+        return isinstance(other, RVoid)
+
+    def __hash__(self) -> int:
+        return 12345
```

Comment:
This looks weird, maybe use something like `hash(RVoid)` (the class object)?

Reply:
Done.